src/parsec_security/secure_deployment.md

```diff
@@ -2,25 +2,31 @@
 operational mitigations. These recommendations have been thought of for a Linux system and should be
-modified/ported for other operating systems. The linked [operational
-mitigations](parsec_threat_model/threat_model.md#operational-mitigations) are noted with `O-n`.
+modified/ported for other operating systems. The Domain Socket Listener and the On-Disk Key Info
+manager are assumed to be used.
+
+The linked [operational mitigations](parsec_threat_model/threat_model.md#operational-mitigations)
+are noted with `O-n`.
 
 - The service must be running as the `parsec` user (O-0).
-- Key mappings must only be read/write by the `parsec` user (O-3).
+- Key mappings must be located at `/var/lib/parsec/mappings` and must only be read/write by the
+   `parsec` user (O-3).
 - The logs must be redirected to a file only readable by the `parsec` user (O-5).
-- The configuration file must only be read/write by the `parsec` user (O-6).
-- For the Domain Socket Listener, the socket must be in a folder with specific permissions (O-9).
-   The folder must be owned by the `parsec` user which must have read, write and execute rights on
-   it.
-   - In a deployment **without Identity Provider**, the folder must be group owned by the
-      `parsec-clients` group which has only read and execute permission on it (O-10). Everyone else
-      must have no permission on the folder. The `parsec-clients` group is composed of mutually
-      trusted clients only.
-   - In a deployment **with Identity Provider**, everyone else can have read and execute permission
-      on the folder.
-- Before accessing the service, clients must check the permissions of the folder containing the
-   socket (O-2). The owner, group owner and permissions must be the same as the previous point,
-   depending on an Identity Provider is available or not.
+- The configuration file must be located at `/etc/parsec/config.toml` and must only be read/write by
+   the `parsec` user (O-6).
+- The socket folder must be located at `/run/parsec/` (O-2) and must only be writable by the
+   `parsec` user (O-9).
+- In a deployment using **Direct authentication with mutually trusted clients**, the socket folder
+   must be group owned by the `parsec-clients` group which has only read and execute permission on
+   it (O-10). Everyone else must have no permission on the folder. The `parsec-clients` group is
+   composed of mutually trusted clients only. This group must be continuously maintained to contain
+   the mutually trusted clients.
+- In a deployment using **Unix Peer Credentials authentication**, everyone else can have read and
+   execute permission on the socket folder.
+- In a deployment using **Unix Peer Credentials authentication**, when a Unix account is deleted,
+   all of its Parsec keys must also be deleted (O-11).
+- The Parsec configuration must not be reloaded with a different authenticator is Parsec is
+   currently storing keys (O-12).
```

Comment:
Maybe add what should be done instead? Like, "remove all contents of `/var/lib/parsec/mappings/`"

Reply:
Ergh, maybe telling people to remove their mappings ain't the best idea. But the point stands - why not or what should they do instead? Maybe use a different folder?

Reply:
Yes good point, I will add that a new folder can be chosen, or all keys should be deleted first.

Reply:
I'm a bit nervous about the "delete all keys" option - should deffo have a "WARNING" in front. ⚠️